src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -428,6 +428,43 @@ private class OffsetAndRule {
             return timeSpans;
         }
 
+        // note the time is already adjusted
+        private AdjustmentRule GetAdjustmentRuleForAmbiguousOffsets(DateTime adjustedTime)
+        {
+            AdjustmentRule rule = GetAdjustmentRuleForTime(adjustedTime);
+            if (rule != null)
+            {
+                if (rule.NoDaylightTransitions && !rule.HasDaylightSaving)
```

Comment:
Just curious: any reason this isn't a single if rather than multiple ifs, e.g.
```C#
if (rule != null && rule.NoDaylightTransitions && !rule.HasDaylightSaving)
{
    ...
}
...
```
?